src/pkg/packager/common.go

```diff
@@ -165,10 +165,10 @@ func createPaths() (paths types.TempPaths, err error) {
 		Base: basePath,
 
 		InjectBinary: filepath.Join(basePath, "zarf-injector"),
-		SeedImage:    filepath.Join(basePath, "seed-image.tar"),
-		Images:       filepath.Join(basePath, "images.tar"),
+		SeedImage:    filepath.Join(basePath, "seed-image"),
+		Images:       filepath.Join(basePath, "images"),
 		Components:   filepath.Join(basePath, "components"),
-		Sboms:        filepath.Join(basePath, "sboms"),
+		SbomTar:      filepath.Join(basePath, "sboms.tar.zst"),
```

Comment:
tar.zst?

Reply:
That's how it was described in the discussion post and I don't think I heard of any other approaches that we want to go with it.

Did  you have something else in mind @Racer159 ?